starter-code/clue.js

```diff
@@ -1,103 +1,87 @@
-
-  Miss Scarlet
-  first name: Kasandra
-  last name: Scarlet
-  color: red
-  description: She is an A-list movie star with a dark past,
-  age: 31,
-  image: https://metrouk2.files.wordpress.com/2016/07/miss-scarlett.jpg,
-  occupation: Actor
-
- Mrs Peacock
-  first name: Eleanor
-  last name: Peacock
-  color: blue
-  description: She is from a wealthy family and uses her status and money to earn popularity
-  age: 36
-  image: https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg
-  occupation: Socialité
-
- Mr Mustard
-  first name: Jack
-  last name: Mustard
-  color: yellow
-  description: He is a former football player who tries to get by on his former glory
-  age: 62
-  image: https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg
-  occupation: Retired Football player
-
----- Weapons ----
-
-Rope - Weight: 10
-Knife - Weight: 8
-Candlestick - Weight: 2
-Dumbbell - Weight: 30
-Poison - Weight: 2
-Axe - Weight: 15
-Bat - Weight: 13
-Trophy - Weight: 25
-Pistol - Weight: 20
-
-
----- Rooms ----
-Dinning Room
-Conservatory
-Kitchen
-Study
-Library
-Billiard Room
-Lounge
-Ballroom
-Hall
-Spa
-Living Room
-Observatory
-Theater
-Guest House
-Patio
-
-*/
+var mrGreen = {
+  firstName: 'Jacob',
+  lastName: 'Green',
+  color: 'green',
+  description: 'He has a lot of connections',
+  age: 45,
+  image: 'https://pbs.twimg.com/profile_images/506787499331428352/65jTv2uC.jpeg',
+  occupation: 'Enterpreneur'
+};
+
+var drOrchid = {
+  firstName: 'Doctor',
+  lastName: 'Orchid',
+  color: 'white',
+  description: 'PhD in plant toxicology. Adopted daughter of Mr. Boddy',
+  age: 26,
+  image: 'http://www.radiotimes.com/uploads/images/Original/111967.jpg',
+  occupation: 'Scientist'
+};
+
+var profPlum = {
+  firstName: 'Victor',
+  lastName: 'Plum',
+  color: 'purple',
+  description: 'Billionare video game designer',
+  age: '22',
+  image: 'https://metrouk2.files.wordpress.com/2016/07/professor-plum.jpg',
+  occupation: 'Designer'
+};
+
+var missScarlet = {
+  firstName: 'Kasandra',
+  lastName: 'Scarlet',
+  color: 'red',
+  description: 'She is an A-list movie star with a dark past',
+  age: '31',
+  image: 'https://metrouk2.files.wordpress.com/2016/07/miss-scarlett.jpg',
+  occupation: 'Actor'
+};
+
+var mrsPeacock = {
+  firstName: 'Eleanor',
+  lastName: 'Peacock',
+  color: 'blue',
+  description: 'She is from a wealthy family and uses her status and money to earn popularity',
+  age: '36',
+  image: 'https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg',
+  occupation: 'Socialité'
+};
+
+var mrMustard = {
+  firstName: 'Jack',
+  lastName: 'Mustard',
+  color: 'yellow',
+  description: 'He is a former football player who tries to get by on his former glory',
+  age: '62',
+  image: 'https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg',
+  occupation: 'Retired Football player'
+};
+
+
+// Deck of cards
+
+var suspects = ['mrGreen','drOrchid','profPlum','missScarlet','mrsPeacock','mrMustard'];
```

Comment:
Watch out! You have to create an array with the variables that you declare before, an array of objects. You are making an array of strings.